python/ray/serve/scripts.py

```diff
@@ -393,7 +429,7 @@ def run(
         sys.exit()
 
 
-@cli.command(help="Get the current config of the running Serve app.")
+@cli.command(help="Gets the current config of the default Serve application.")
```

Comment:
```suggestion
@cli.command(help="Gets the current config of a Serve application.")
```

It looks like it can get the config of more than just the default.

Reply:
I wrote it this way because I wanted to make it clear that the default behavior of the CLI is to grab the default app with name="" - there doesn't seem to be a better place to describe this to the user. Or, do you think it's better to describe these details in the description of `--all-apps`?

Reply:
Yeah, I'd recommend putting this in the description of `--all-apps` or `--app-names`. Does this command fail if the user is using multi-app but doesn't set any of the flags?

Reply:
No, currently we don't track in the backend whether user is using multi-app or not. I'll be adding this in a PR soon. And that works, I'll tweak the descriptions a bit to try to make it clear to the user.

Reply:
Gotcha, so if a user is using multi-app and runs `serve config` with no flags, they'll get their original config?

Reply:
Ah I see what your original question meant - so if users deploy a multi-app config and run `serve config`, they likely won't get the output they want, because `serve config` with no flags will try to get the config for the application with name="". In almost all cases this application would not exist (I can't really see users intentionally deploying an application with an empty string as the name), so it would show the `empty_schema_dict` like:
```
import_path: ''
runtime_env: {}
deployments: []
```

Reply:
I see, that makes sense.

> currently we don't track in the backend whether user is using multi-app or not. I'll be adding this in a PR soon.

After adding this PR, would it be possible to create a more intuitive output (such as logging a warning)?

Reply:
Yup, I'll include that!

Reply:
Sounds good, thanks!